assembly/package-res/biserver/pentaho-solutions/system/pentaho.xml

```diff
@@ -27,6 +27,8 @@
 	<log-file>server.log</log-file>
 	<log-level>DEBUG</log-level>
 
+	<verify-user-on-emitting-principal>false</verify-user-on-emitting-principal>
```

Comment:
I want this to be "true" by default. So that users using their old pentaho.xml will get this new behavior by default.

Change this to "true", and please change the name to "verify-user-on-principal-creation". Add a comment above describing the setting and what happens if set to "false"